components/pricing.tsx

```diff
@@ -45,28 +123,38 @@ const PricingTier: React.FC<PricingTierProps> = ({
           </p>
         )}
         {isFree && (
-          <p className="text-sm font-medium text-gray-400 sm:text-base">
-            <a
-              href="https://forms.gle/171UyimgQJhEJbhU7"
-              className="text-link"
-              target="_blank"
-            >
-              Join the waitlist
-            </a>{" "}
-            to be notified when the app is available!
-          </p>
+          <>
+            <p className="text-sm font-medium text-gray-400 sm:text-base">
+              <a
+                href="https://forms.gle/171UyimgQJhEJbhU7"
+                className="text-link"
+                target="_blank"
+              >
+                Join the waitlist
+              </a>{" "}
+              to be notified when the app is available!
+            </p>
+            {!user && (
+              <p>
+                Please{" "}
+                <a href="/signin" className="text-link">
+                  log in
+                </a>{" "}
+                to download for free.
+              </p>
+            )}
+          </>
```

Comment:
should we login even to download for free?

Reply:
^ Yes correct, we shouldn't

Reply:
Yes for now! (due to waitlist)